load.php

```diff
@@ -39,3 +40,6 @@
 
 	register_module( 'cloud', __DIR__, 'Cloud', $default_settings, __NAMESPACE__ . '\\bootstrap' );
 } );
+
+// Early hook for logging AWS SDK HTTP requests.
+add_filter( 'altis.aws_sdk.params', __NAMESPACE__ . '\\add_aws_sdk_xray_callback' );
```

Comment:
We need to separately hook into S3 Upload's copy of the SDK currently.

```suggestion
add_filter( 'altis.aws_sdk.params', __NAMESPACE__ . '\\add_aws_sdk_xray_callback' );
add_filter( 's3_uploads_s3_client_params', __NAMESPACE__ . '\\add_aws_sdk_xray_callback' );
```